quiz-service/quiz-infra/src/main/kotlin/com/grepp/quizy/quiz/infra/comment/repository/CommentRepositoryAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package com.grepp.quizy.quiz.infra.comment.repository
+
+import com.grepp.quizy.quiz.domain.comment.Comment
+import com.grepp.quizy.quiz.domain.comment.CommentId
+import com.grepp.quizy.quiz.domain.comment.CommentRepository
+import com.grepp.quizy.quiz.domain.quiz.QuizId
+import com.grepp.quizy.quiz.infra.comment.entity.CommentEntity
+import org.springframework.stereotype.Repository
+import org.springframework.transaction.annotation.Transactional
+
+@Repository
+@Transactional
+class CommentRepositoryAdapter(
+        private val commentJpaRepository: CommentJpaRepository
+) : CommentRepository {
+
+    override fun save(comment: Comment): Comment =
+            commentJpaRepository.save(CommentEntity.from(comment)).toDomain()
+
+    override fun findById(id: CommentId): Comment? =
+            commentJpaRepository
+                    .findById(id.value)
+                    .map { it.toDomain() }
+                    .orElse(null)
```

Comment:
이거 되는지 확신이 안서긴하는데 이런게 있더라구요? 한번 되는지 확인해보고 바꾸시면 좋을 것 같아요 이게 훨씬 편하더라구요
```suggestion
    override fun findById(id: CommentId): Comment? =
            commentJpaRepository
                    .findByIdOrNull(id.value)?.toDomain()
```

`findByIdOrNull`이라는게 있어서 더 간단하게 쓸 수 있는거같더라구요 한번 확인해보시면 좋을 것 같아요